code/exception/handler/abstract.php

```diff
@@ -354,6 +354,37 @@ public function isEnabled($type)
     }
 
     /**
+     * Exception Handler
+     *
+     * Do not call this method directly. Function visibility is public because set_exception_handler does not allow for
+     * protected method callbacks.
+     *
+     * @param  object $exception  The exception to be handled
+     * @return bool
+     */
+    public function _handleException($exception)
+    {
+        $result = false;
+
+        if($this->isEnabled(self::TYPE_EXCEPTION))
+        {
+            //Handle \Error Exceptions in PHP7
+            if (class_exists('Error') && $exception instanceof \Error)
+            {
+                $message = $exception->getMessage();
```

Comment:
The problem is that in that case I need to remove all \Exception checks in the code, everywhere we handle errors. I'm trying to prevent that as this fix needs to be BC. 

The stack trace issue is not really a problem, our debugger can render full stacktraces  if xdebug is on.